install.sh

```diff
@@ -68,3 +68,17 @@ fi
 
 # Restarting service
 service dnsmasq restart
+
+# Automatically login from clients 
+if [[ ${AUTOMATIC_LOGIN,,} == "yes" ]]; then
+    sed -i /var/lib/tftpboot/ltsp/i386/lts.conf \
```

Comment:
Please check `ltsp-config` output, it give you which file it had created and use it here.

Reply:
For example
```
akash@skyfall:/var/lib/tftpboot/ltsp$ sudo ltsp-config -o lts.conf|cut -d' ' -f2
/var/lib/tftpboot/ltsp/amd64/lts.conf

```

Reply:
Right. `ltsp-config` returns lts.conf path

Reply:
Also it will handle both cases `i386` and `amd64`
clever :+1: